Tasks/NuGetInstaller/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Agent.Tasks",
```

Comment:
No reason to add this file.  I'm adding support and will add this package.json for you.  Also, the name should be your task, not Agent.Tasks